bundles/org.openhab.binding.boschshc/src/test/java/org/openhab/binding/boschshc/internal/devices/bridge/LongPollingTest.java

```diff
@@ -237,7 +237,7 @@ void start() throws InterruptedException, TimeoutException, ExecutionException,
         verify(longPollHandler).accept(longPollResultCaptor.capture());
         LongPollResult longPollResult = longPollResultCaptor.getValue();
         assertEquals(1, longPollResult.result.size());
-        DeviceServiceData longPollResultItem = longPollResult.result.get(0);
+        DeviceServiceData longPollResultItem = (DeviceServiceData) longPollResult.result.get(0);
```

Comment:
Maybe i'm missing something, but this cast was not neccesary before, so why add it?

Reply:
Before my change the longPollResult only consisted of DeviceServiceData objects. With my changes you may now get a DeviceServiceData or Scenario back. Thats why I had to explicit cast the result in this test.

Reply:
Oke, if you are sure it will allways return a `DeviceServiceData`, oterwise i would suggest something with instanceof or getClass

Reply:
This test always returns the following mocked json response which is a `DeviceServiceData`:
`String longPollResultJSON = "{\"result\":[{\"path\":\"/devices/hdm:HomeMaticIP:3014F711A0001916D859A8A9/services/PowerSwitch\",\"@type\":\"DeviceServiceData\",\"id\":\"PowerSwitch\",\"state\":{\"@type\":\"powerSwitchState\",\"switchState\":\"ON\"},\"deviceId\":\"hdm:HomeMaticIP:3014F711A0001916D859A8A9\"}],\"jsonrpc\":\"2.0\"}\n";`